src/oop/app/src/Parsers/KinoukrDomCrawlerParserAdapter.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace src\oop\app\src\Parsers;
+
+use src\oop\app\src\Models\Movie;
+use Symfony\Component\DomCrawler\Crawler;
+
+class KinoukrDomCrawlerParserAdapter implements ParserInterface
+{
+    private Crawler $crawler;
+    private Movie $movie;
+
+    public function __construct(){
+        $this->crawler = new Crawler();
```

Comment:
please do not use new keyword, use dependency injection:
public function __construct(Crawler $crawler, Movie $movie)
{
$this->crawler = $crawler;
...